### PR TITLE
fix 519

### PR DESCRIPTION
**Pull request by tatchi**
This fixes the remaining failing test (519).

### CODE REVIEW

File: src/parser.ml [hunk 1]
```diff
@@ -1719,13 +1719,13 @@ let rec inline defs st =
     | exception Fail ->
         junk st;
         loop (Left_bracket kind :: text acc) st
-  and loop acc st =
+  and loop ~seen_link acc st =
```
Comment (tatchi) [inline src/parser.ml:1722]: We can avoid passing this `seen_link` argument every time by storing of the `Url` delimiter `[` or `]` is active or not. That's what I did in this branch: https://github.com/ocaml/omd/compare/master...fix-519

Comment (tatchi) [reply src/parser.ml:1722]: Let me know which solution do you prefer :)

Comment (shonfeder) [reply src/parser.ml:1722]: I'm ok with this approach! I assume you also prefer it since it's what you used for the PR :)

**Review by shonfeder — APPROVED** [review]

**Comment by shonfeder** [comment]
Amazing. Thank you for the wonderful work dispatching all the spec exceptions, @tatchi!

I think one thing we've omitted in the last PR's is an update to the changelog that records these fixes, and celebrates your responsibility for them :)

If you'd like to add a PR memorializing your fixes in a changelog entry, please do. Otherwise I'll take care of adding that this weekend. 

:tada:

**Pull request merged**